modules/brightcomSSPBidAdapter.js

```diff
@@ -319,4 +324,72 @@ function _getBidFloor(bid) {
   return null;
 }
 
+function _getFirstPartyData() {
```

Comment:
don't copy paste huge code blocks into two adapters, use a library

Reply:
hi @patmmccann ! I'm not quite sure how libraries are organized in this project. Could you share a few examples with me? I'd greatly appreciate it!

thanks!

Reply:
https://github.com/prebid/Prebid.js/blob/master/libraries/keywords/keywords.js is a code block repeated across multiple adapters. One of our goals to reduce build size is to reduce copy pasted code blocks. 

![image](https://github.com/prebid/Prebid.js/assets/1683175/5a593a6b-e762-4268-b180-d226fa10fbde)